storage/database/db_manager.go

```diff
@@ -478,6 +522,39 @@ func (dbm *databaseManager) SetStateTrieMigrationDB(blockNum uint64) {
 		logger.Crit("Killed the process due to the failure of storing the migration status")
 	}
 	logger.Info("Finished setting a new database for state trie migration")
+	return nil
+}
+
+// FinishStateMigration updates stateTrieDB and triggers to remove old one.
+func (dbm *databaseManager) FinishStateMigration() {
+	oldDBDir := dbm.getDBDir(StateTrieDB)
+	newDBDir := dbm.getDBDir(StateTrieMigrationDB)
+	dbm.setDBDir(StateTrieDB, newDBDir)
+
+	oldDB := dbm.dbs[StateTrieDB]
+	dbm.dbs[StateTrieDB] = dbm.dbs[StateTrieMigrationDB]
+
+	dbm.inMigration = false
+
+	// TODO-Klaytn Refine remove DB code
+	go func(dbDir string) {
+		// TODO-Klaytn remove oldDB after StakingInfoInterval
+		time.Sleep(10 * time.Second)
+		oldDB.Close()
+		if err := os.RemoveAll(dbDir); err != nil {
+			logger.Error("Failed to remove the state trie migration database due to an error", "err", err, "dir", dbDir)
+			return
+		}
+		logger.Info("Successfully removed the state trie migration database", "dir", dbDir)
+	}(oldDBDir)
+}
+
+func (dbm *databaseManager) GetOldStateTrieDB() Database {
+	return dbm.dbs[StateTrieDB]
+}
+
+func (dbm *databaseManager) GetNewStateTrieDB() Database {
```

Comment:
```suggestion
func (dbm *databaseManager) GetInMigrationStateTrieDB() Database {
```

Reply:
I reflected in https://github.com/klaytn/klaytn/pull/502